<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор псевдовипадкових чисел</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>851</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>-150</x>
       <y>10</y>
       <width>401</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Основне меню</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Параметри генератора</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>від</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>до</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>від</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ГПВЧ на основі МСК</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Лінійний конгруентний метод</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Алгоритм Лемера</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Метод Фібоначчі з запізненнями </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Квадратичний конгруентний метод</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Мультиплікативний метод</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Лінійний метод</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Вихор Мерсенна</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Xorshift32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marsaglia-multicarry</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Згенерувати</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Кількість значень</string>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>comboBox</zorder>
     <zorder>pushButton</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>spinBox</zorder>
     <zorder>spinBox</zorder>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерація випадкових значень</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенеровані значення</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
